app/views/categories/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<nav>
+  <div><h2>Categories</h2></div>
+  <%= button_to 'Sign Out', destroy_user_session_path, method: :delete %>
+</nav>
+<section class="main">
+  <% @categories.each do |category| %>
+    <section class="category">
+      <div>
+        <img src="<%= category.icon %>">
+        <%= link_to category_procurements_path(category) do %>
+          <%= category.name %>
+        <% end %>
+      </div>
+      <% procurement_sum = 0 %>
+      <% category.procurements.each do |procurement| %>
+        <% procurement_sum += procurement.amount %>
+      <% end %>
+      <div>
+        <p>$<%= procurement_sum %></p>
+      </div>
+    </section>
+  <% end %>
+</section>
+<div class="add-btn btn">
+  <%= link_to 'Add a New Category', new_categories_path %>
+</div>
```

Comment:
| @Wineshuga Design |
|-|
| ![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/b659b424-bdff-4f25-8c46-a286c5f7983f) |

| Microverse Design |
|-|
| ![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/32db909c-460a-49b4-b556-653ab4a9feac) |

- [ ] It would be nice if you made your button look similar to the given design because the white border almost touches the edges of the green borders 🤔 to make your design look professional 👍 Your white border is touching the upper and lower edges so you just need to make it touch the left and right edges 😊